src/sugar3/graphics/radiopalette.py

```diff
@@ -76,10 +97,17 @@ def append(self, button, label):
             self.__clicked_cb(button)
 
     def update_button(self):
+        '''
+        Updates the button
```

Comment:
What does this mean?  Does it spin it in a circle?  Does it change everything to use cat pictures?